docs/guides/qiskit-addons-mpf-get-started.ipynb

```diff
@@ -0,0 +1,372 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0d234a52-9654-4ee1-bb23-f6705154c3b0",
+   "metadata": {},
+   "source": [
+    "## Getting started with Multi-product formulas (MPF)\n",
```

Comment:
```suggestion
    "## Get started with Multi-product formulas (MPF)\n",
```
Is Multi-product formulas considered a "product" name? if so, we can keep the capital M here. Otherwise, lower-case m.